### ISSUE TITLE
App crashes immediately upon opening (C2-12828-1)

### ISSUE THREAD

**Issue by Iristallite** [devnoname120/vitabright-lut-editor#6]
I tried to open this app, but it just crashes with the error C2-12828-1.
My console is a PCH-1001, running an sd2vita with a 128GB microSD card.
I have vitabright v2.0 installed.

I've attached the crashdumps and my tai config.txt, in case they could help.
crashdumps: [crashdumps.zip](https://github.com/devnoname120/vitabright-lut-editor/files/8306360/crashdumps.zip)
tai config.txt: [config.txt](https://github.com/devnoname120/vitabright-lut-editor/files/8306359/config.txt)

**Comment by Iristallite** [comment]
Tried loading it with ALL my plugins disabled, except for vitabright, and I also disabled my sd2vita entirely, but it still crashed...
crashdump: [crashdump2_noplugins_except_vitabright.zip](https://github.com/devnoname120/vitabright-lut-editor/files/8306436/crashdump2_noplugins_except_vitabright.zip)

**Comment by devnoname120** [comment]
Does it work if you disable `NoPowerLimits.skprx`?

If not, please disable all your plugins except vitabright and try again.

**Comment by buzeak** [comment]
@Nordii08 When you say "Crashes immediately upon opening" are you trying to open the Vitabright lut editor app? Or does your vita not boot up?

**Comment by buzeak** [comment]
@Nordii08 You could give me your latest tai config.txt with all plugins disabled, and I will test it on my PCH-1003.

**Comment by buzeak** [comment]
@Nordii08 You could try this empty exept for vitabright: [config.txt](https://github.com/devnoname120/vitabright-lut-editor/files/8306837/config.txt)

It may mess up the location of the apps on your homescreen though. (it did with mine)

**Comment by buzeak** [comment]
@devnoname120 @Nordii08 I have just found that the live lut editor app now crashes on edit (as soon as you press up or down). So I'm guessing that this is the problem in this issue.

**Comment by Iristallite** [comment]
> Does it work if you disable `NoPowerLimits.skprx`?
> 
> If not, please disable all your plugins except vitabright and try again.

I tried with every single plugin (except vitabright.skprx) disabled, and it still crashed

> @Nordii08 When you say "Crashes immediately upon opening" are you trying to open the Vitabright lut editor app? Or does your vita not boot up?

The editor app is where it crashes. My vita boots up fine, and the plugin itself does work.

> @Nordii08 You could try this empty exept for vitabright: [config.txt](https://github.com/devnoname120/vitabright-lut-editor/files/8306837/config.txt)
> 
> It may mess up the location of the apps on your homescreen though. (it did with mine)

Nope, still crashes with that config

**Comment by buzeak** [comment]
@devnoname120 @Nordii08 Yes, it is working with [vitabright-2.0b2.zip](https://github.com/devnoname120/vitabright-lut-editor/files/8307093/vitabright-2.0b2.zip)

But the latest vitabright.skprx is not compatible with the live lut editor app.

**Comment by buzeak** [comment]
@devnoname120 Will hopefully be able to fix this new issue @Nordii08, once he has a look.

**Comment by devnoname120** [comment]
I think I kind of know what's the issue. I'll fix it in 1-2 weeks I'm busy right now and it's not a priority since you can just use the 2nd beta that @buzeak linked.

**Comment by Iristallite** [comment]
Just to clarify, my issue is that the vitabright-lut-editor app crashes immediately after I open it. The plugin works fine, and my vita has no problem booting

(I say this because I couldn't tell if everyone here understood properly. I hope I don't sound rude, and if I do, then I'm very sorry.)
And no, using that 2.0b2 version did *not* fix the issue.

**Comment by buzeak** [comment]
@Nordii08 @devnoname120 I have found that [vitabright-lut-editor-1.1.vpk](https://github.com/devnoname120/vitabright-lut-editor/releases/download/1.1/vitabright-lut-editor-1.1.vpk) (471,886 bytes) crashes on opening and I don't know why, and the [vitabright-lut-editor.zip](https://github.com/devnoname120/vitabright-lut-editor/files/8307799/vitabright-lut-editor.zip) (469,677 bytes) I've been using only works with [vitabright-2.0b2.zip](https://github.com/devnoname120/vitabright-lut-editor/files/8307806/vitabright-2.0b2.zip)

I can't remember where I got my version of the lut-editor from, but it looks like an older model.

So please try the working combination above Nordii :-)

**Comment by tonyzehs** [comment]
I had the same problem with the 2.0 VitaBright, but adding an empty line below the vitabright_lut.txt made it launch successfully. While it can display things, making any change will instantly freeze the system until force reboot to safe mode.

**Comment by tonyzehs** [comment]
> So please try the working combination above Nordii :-)

With the above combination it still crashes and restarts upon opening but adding an empty line at the very bottom of the vitabright_lut.txt made it launch successfully. Edits are also working now.

**Comment by Ivirawr** [comment]
I had the same problem too, with vitabright 1.0, i can lunch the Lut editor, but doesn't do anything

**Comment by devnoname120** [comment]
@Ivirawr Can you try to retransfer the original LUT via USB? (FTP can fuck up line endings.)

**Comment by Ivirawr** [comment]
> 
lut_orig.txt?

**Comment by Ivirawr** [comment]
[error.zip](https://github.com/devnoname120/vitabright-lut-editor/files/9149208/error.zip) This is what i get when i open the lut editor

**Comment by meeeeeeeeeeeeeeee** [comment]
I was having the same problem on my PCH-1001 (WiFi Only) Vita running 3.60 with enso, changes I made to the lut table worked but the app would not open and I've had this problem since 2.0 came out. 

I fixed it by moving the app to my official memory card (I moved the app to internal storage with application storage manager) then I deleted the lut table the plugin file and the app and rebooted my vita, while it was booting I used emergency mount to make ur0 show up on my computer over USB and installed vitabright 1.1 then I mounted ux0 the same way and copied the vpk for the app to the vpk folder and hit exit in emergency mount, when the vita booted I installed the app in vitashell then launched it and it worked but the screen was orangey at lower brightness levels because I copied 1.1's lut table instead of keeping 2.0's, to fix this all I had to do was boot into emergency mount again and mount ur0 then I downloaded vitabright 2.0 and duplicated the "vitabright_lut.txt" file 2.0 comes with and renamed the duplicate to "vitabright_lut_orig.txt" then I copied both files over the older versions from 1.1 and the app still launched after rebooting again.

**Comment by nezumisama** [comment]
I have PCH-1000 Wifi OLED Vita with sd card (via sd2vita) with FW 3.60, henkaku, enso and storage manager. I had the same issue as OP, and buzeak's suggestion above worked for me, except I have the latest vitabright 2.0, not the beta. I just installed the version of lut editor from the zip instead of the latest release.

**Comment by 0lorak** [comment]
In my case, _vitabright-lut-editor-1.1.vpk_ keps crashing just after launching it. However, _vitabright-lut-editor.vpk_ that @buzeak shared above works ok with both plugins, _vitabright2.0_ and _vitabright2.0b2_. I do not even need to add a blank line at the end of vitabright_lut.txt.

@buzeak and @devnoname120: What are you adding in _vitabright2.0b2_? Do you recommend 2.0 or 2.0b2?
